Documentation/Settings.cfg

```diff
@@ -1,39 +1,62 @@
-[general]
-copyright = 2016
-project = TYPO3 Console
-release = 7.1.1
-version = 7.1.1
+# More information about this file:
+# https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#settings-cfg
 
-[html_theme_options]
-github_branch = latest
-github_commit_hash =
-github_repository = TYPO3-Console/TYPO3-Console
-path_to_documentation_dir =
-github_revision_msg =
-github_sphinx_locale =
-project_contact =
-project_discussions =
-project_home =
-project_issues = https://github.com/TYPO3-Console/TYPO3-Console/issues
-project_repository = https://github.com/TYPO3-Console/TYPO3-Console
-use_opensearch =
+[general]
 
-[intersphinx_mapping]
-t3editors = https://docs.typo3.org/typo3cms/EditorsTutorial/
-t3start = https://docs.typo3.org/typo3cms/GettingStartedTutorial/
-t3tsref = https://docs.typo3.org/typo3cms/TyposcriptReference/
+project     = TYPO3 Console
+version     = main (development)
+release     = main (development)
```

Comment:
What are the exact expectations / requirements for these version/ release values now.

Setting it to a string, will break my release workflow.

Reply:
The documentation of the main branch should contain `main (development)` and all release branches should contain the minor release version, e.g. `6.7` for the `6.x`.

Using the release version in the main branch is a bit wrong, as docs.typo3.org shows really the main branch tip when selecting `main` in the version select on the left side. But as your 7.x versions are contained in the main branch, i guess it would be correctest, if you switch from `main (development)` to `7.<x>` for each release commit and then switch back. Maybe @marble has a strong opinion here?

Reply:
OK. I see two options to proceed. Revert the versioning change here and merge it, or add code to support a new release workflow first. 

I'd prefer the first, as I'm in general not keen on touching the release workflow, although I agree it is slightly incorrect regarding versioning of the tool and the documentation.

Reply:
Yes, i agree to revert the versioning and we will discuss the topic in the documentation team. Greetings, alex

Reply:
Thanks. Pushed a commit that does that and also fixes the script to change the version number